### PR TITLE
FIX: Make 'Choose Report Type' Dropdown list GitHub repo's issue template, update issue body with template content

### PR DESCRIPTION
**Pull request by KBandipo**
#### This pull request makes the following changes

* Fixes collabo-community/issue-tickets-ready-for-fixing#37

#

#### General checklist

- [x] My pull request adheres to the [Collabo Community pull request guidelines](https://docs.collabocommunity.com/pull-request-guidelines).
- [x] I have read and understood the **Pull request guideline extension** section found at the bottom of this pull request description/body.
- [x] I have replaced the dummy text under the subheading **Testing checklist** below with the testing checklist from the issue ticket I'm working on.

#

#### Testing checklist

- [x] 'Choose Report Type' Dropdown now lists issue templates from connected GitHub repo by name.
- [x] Selecting/changing a template name from the 'Choose Report Type' dropdown list now updates the issue body form input of the modal, with the appropriate template content accordingly.
- [x] I certify that I ran my checklist

#

#### Pull request guideline extension

> [!NOTE]  
> Do not remove this note or section. This talks about extra things to take note of, in addition to the directions given in the Collabo Community pull request guidelines.

We appreciate the time taken to help tackle issue tickets within the Collabo Community. Please go ahead to submit your pull request even if it is not yet completed, and push to the pull request actively. This will allow us to know that you have not abandoned the issue ticket. However, you should only alert reviewers about your pull request as explained below, to avoid flooding their GitHub notifications unnecessarily.

"Ready for review" implies that things are working as expected, according to how it is specified in the issue ticket you are working on. Ensure that you test the code that you are working on, that it works as expected. Don't just request for review when your pull request is not yet done and working.

The only time you should be making people review something incomplete is: if you want to show them what's not working, so that they can send help or guidance. If this is the case, be explicit about this in our Discord community or in the comment of your pull request (after you have submitted the pull request).

### CODE REVIEW

**Comment by KBandipo** [comment]
Ready for review

**Review by samuko-things — APPROVED** [review]
Good Job @KBandipo, I have no issues with the code and I have also tested it.

File: @dev-client/views/@plugin_githubsync/githubsync.js [hunk 1]
```diff
@@ -48,6 +48,57 @@ if (response.status === 401){
...
+    console.log('Error fetching templates:', error);
+    displayToastrMessage.innerHTML = 'Could not load issue templates.';
+    return [];
+  }
+};
+
+const fetchTemplatesDropdown = async () => {
+  const templates = await fetchTemplates();
+  templates.forEach((template) => {
+    const option = document.createElement('option');
```
Comment (Ifycode) [inline @dev-client/views/@plugin_githubsync/githubsync.js:69]: Thank you @samuko-things 
Just one thing @KBandipo, let's use `template literal` to create elements in the javascript instead.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: @dev-client/views/@plugin_githubsync/githubsync.js (hunk 1 above)
Comment (KBandipo) [reply @dev-client/views/@plugin_githubsync/githubsync.js]: Okay,copy

**Comment by KBandipo** [comment]
Ready for Review

**Review by Ifycode — APPROVED** [review]

**Pull request merged**